README.md

```diff
@@ -1,70 +1,19 @@
-# Getting Started with Create React App
+## Track Hub Registry (frontend)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+### Prerequisites
 
-## Available Scripts
+* [node 16+](https://nodejs.org/en/download/)
 
-In the project directory, you can run:
+### Local deployment
 
-### `npm start`
+Running the frontend for the first time:
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+```
+git clone https://github.com/Ensembl/thr_frontend.git
+cd thr_frontend
+npm install -g create-react-app # You may need 'sudo'
```

Comment:
Why does this need to be global?

Reply:
You're right, actually, there is no need for that command at all since we are cloning the project and not creating it from scratch